README.md

```diff
@@ -67,15 +77,38 @@ In Swift / Objective-C, Babbage must be initialized with two arguments. The firs
 // In AppDelegate.swift
 let dir = NSFileManager.defaultManager().URLsForDirectory(NSSearchPathDirectory.DocumentDirectory, inDomains: NSSearchPathDomainMask.UserDomainMask).last
 let key = "12345678901234567890123456789012".dataUsingEncoding(NSUTF8StringEncoding)
-MDBabbage.startWithDatastoreAtURL(dir, encryptionKey: key)
+MDBabbage.startWithEnvironment(MDClientEnvironment.Sandbox, apiToken: "Your provided API token", publicDirectory: dir, privateDirectory: dir, encryptionKey: key)
+```
+
+## Self Registering the user
+You can create an account for storing data to AWS S3 buckets.
+
+```swift
+// In CreateAccountViewController.swift
+let userEmail = "unique email address"
+let userPassword = "password complying to have  
+•  At least 8 characters long
+•  At least one upper-case letter
+•  At least one lower-case letter
+•  At least one numeric digit"
+let userSecurityQuestionID = "id of the array element from list of questions"
+let userSecurityAnswer = "answer to the security question"
+
+client.registerSubjectWithEmail(userEmail, password: userPassword, securityQuestionID: userSecurityQuestionID, securityQuestionAnswer: userSecurityAnswer completion:(void (^)(NSError *error))completion
 ```
 
-## Loading Data from the Datastore
+## Loading Data to the Datastore
```

Comment:
I don't think this title could be a bit confusing. It looks like we have a TODO in the `Upload Data to S3` section, maybe it would be better under that header?